Models/Item.cs

```diff
@@ -1,11 +1,13 @@
-﻿using System.ComponentModel.DataAnnotations.Schema;
+﻿using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
 
 namespace HowManyCaloriesIn.Models
 {
-	[Table("tblItem")]
-    public class Item
+	[Table("Item")]
+    public partial class Item
```

Comment:
Is there a reason why this is a partial class? I don't think you're making use of the partial modifier?